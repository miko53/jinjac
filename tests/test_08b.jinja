Display of array direclty
double: {{ data_value_dbl }}
string: {{ users }}
int: {{ data_value }}
---
