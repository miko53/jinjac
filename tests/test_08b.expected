Display of array direclty
double: [0.586000, 10.450000, 159.546000, 3.145561]
string: [u'dana', u'bob', u'john']
int: [10, 20, 15]
---
